themes/default/content/product/_index.md

```diff
@@ -12,80 +12,115 @@ subheading: |
                 icon_color: salmon
                 description: |
-                    Run deployments from your application code at runtime with [Automation API](/automation/). Create infrastructure APIs, custom platforms, and CLIs.
+                    Develop modern cloud applications faster while using well-architected best practices with the Crosswalk for AWS library.
 
-              - title: Preview and test changes
+              - title:
                 icon: eye
                 icon_color: salmon
                 description: |
-                    Test and validate infrastructure with standard [unit test frameworks](/docs/guides/testing/#unit-testing) and
-                    [integration tests](/docs/guides/testing/integration/). Preview changes before deploying.
+                    Convert your existing infrastructure code to Pulumi (TF, ARM, CFN, K8s, CRD, Helm) and import existing resources into Pulumi code.
+     
+        - description: Deploy to over 100 cloud and SaaS providers
+          features:
+              - title: 
+                icon: shield
+                icon_color: purple
+                description: |
+                    Native providers for AWS, Azure, Google Cloud, and Kubernetes have 100% API coverage and same-day updates for new features.
+
+              - title: 
+                icon: security
+                icon_color: purple
+                description: |
+                    Use a single workflow to manage cloud native infrastructure like Kubernetes, Helm, CRDs, GitOps, and more.
 
-        - description: Manage cloud applications with visibility and controls
+              - title: 
+                icon: monitor
+                icon_color: purple
+                description: |
+                    Find 100+ providers and cloud components in the Pulumi Registry. Bridge any existing Terraform provider to Pulumi.
+
+        - description: Not just a CLI, but Infrastructure as Software.
           features:
-              - title: Policy as Code
+              - title: Pulumi CLI (open source)
                 icon: shield
                 icon_color: purple
                 description: |
-                    [Enforce compliance and detect drift](/crossguard/) by checking infrastructure against rules for
-                    security, cost, and best practices.
+                    Run IaC actions from your workstation with the open source command line interface.
 
-              - title: State and secrets management
+              - title: Pulumi as a Library (open source)
                 icon: security
                 icon_color: purple
                 description: |
-                    [Pulumi Service](/product/pulumi-service/) manages your infrastructure state and automatically
-                    encrypts sensitive values in transit and at rest.
+                    Program IaC actions into your application code with the open source Automation API library.
```

Comment:
link Automation API library to https://www.pulumi.com/docs/guides/automation-api/